ansible/roles/common/templates/sshd_config

```diff
@@ -0,0 +1,77 @@
+Port {{ liquid_ssh.port }}
+#ListenAddress ::
```

Comment:
Really, let's lose all the comments, they are just noise. We can always look up sshd configuration options in the man page.